app/views/components/_reorderable_list.html.erb

```diff
@@ -0,0 +1,39 @@
+<%
+  items ||= []
+  data_attributes ||= {}
+  data_attributes[:module] = "reorderable-list"
+  input_name ||= "reorderable-list-#{SecureRandom.hex(4)}"
```

Comment:
One other suggestion that has just occurred to me - should the component take an `error_message`/`error_items` property like [other components in govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_input.html.erb)?

I imagine if a user enters an unparseable input on one of the inputs, we should highlight the specific input in a red error state and explain what is wrong.

I'm a little hesitant to suggest this, as I know @benthorner was keen we don't bog ourselves down in validation. I suppose the alternative would be to just skip over any offending inputs and put them at the back of the queue, so to speak, though once we start adding that logic we're not far off the complexity we'd get from adding 'proper' validation.

Reply:
Good question about this. I didn't want to introduce it in the component before discussing how/if we're going to support this.